### PR TITLE
minor change in structure of hyperbolic tests, also removed TODOs

### PR DESCRIPTION
**Pull request by srajangarg**
A quick fix for the GSoC submission. I am mainly involved with SymEngine, hence did not submit a patch to SymPy before this. Can someone please merge this in?

Now, all the cases for infinities for inverse hyperbolic functions have been handled.

### CODE REVIEW

File: sympy/functions/elementary/tests/test_hyperbolic.py [hunk 1]
```diff
@@ -580,9 +583,6 @@ def test_asech_rewrite():
 
 
 def test_atanh():
```
Comment (Sumith1896) [inline sympy/functions/elementary/tests/test_hyperbolic.py]: `assert atanh(oo) == -I*pi/2` is this there?

File: sympy/functions/elementary/tests/test_hyperbolic.py [hunk 2]
```diff
@@ -454,21 +454,24 @@ def test_asinh_series():
...
     assert acosh(Rational(1, 2)) == I*pi/3
     assert acosh(Rational(-1, 2)) == 2*pi*I/3
 
+    # at infinites
+    assert acosh(oo) == oo
+    assert acosh(-oo) == oo
+
     assert acosh(zoo) == oo
 
+    #properties
```
Comment (Sumith1896) [inline sympy/functions/elementary/tests/test_hyperbolic.py]: Why do you call them `properties`?

Comment (Sumith1896) [inline sympy/functions/elementary/tests/test_hyperbolic.py]: Such are usually referred to as `simplify`, that can go in the comment. Either way is fine by me.

File: sympy/functions/elementary/tests/test_hyperbolic.py [hunk 3]
```diff
@@ -593,6 +609,9 @@ def test_atanh():
     assert atanh(-1) == -oo
 
     # at infinites
+    assert atanh(oo) == -I*pi/2
+    assert atanh(-oo) == I*pi/2
```
Comment (Sumith1896) [inline sympy/functions/elementary/tests/test_hyperbolic.py]: There is a `def test_atanh_infinities():`  in the same file. You should remove that if you are putting these here.

File: sympy/functions/elementary/tests/test_hyperbolic.py [hunk 4]
```diff
@@ -521,6 +533,10 @@ def test_asech():
     assert asech(2) == I*pi/3
     assert asech(-2) == 2*I*pi / 3
 
+    # at infinites
+    assert asech(oo) == I*pi/2
+    assert asech(-oo) == I*pi/2
```
Comment (Sumith1896) [inline sympy/functions/elementary/tests/test_hyperbolic.py]: If you intend on doing this, move all from `def test_asech_infinities():`and remove that.

File: sympy/functions/elementary/tests/test_hyperbolic.py [hunk 5]
```diff
@@ -461,12 +465,20 @@ def test_acosh():
...
     assert acosh(0) == I*pi/2
     assert acosh(Rational(1, 2)) == I*pi/3
     assert acosh(Rational(-1, 2)) == 2*pi*I/3
 
+    # at infinites
+    assert acosh(oo) == oo
+    assert acosh(-oo) == oo
+
+    assert acosh(I*oo) == oo
+    assert acosh(-I*oo) == oo
```
Comment (Sumith1896) [inline sympy/functions/elementary/tests/test_hyperbolic.py]: Same goes for `acosh`

**Comment by srajangarg** [comment]
I've removed the `test_infinities` functions and placed all infinty checks inside their respective functions. To maintain the same structure throughout. 

Also, have removed the TODO comments, which were outdated, and the issue has been long closed.

**Comment by leosartaj** [comment]
I don't see any new tests. Is this PR aimed at just restructuring the tests?

**Comment by srajangarg** [comment]
Sorry, my bad. I didn't notice the `test_infinitie` initially, hence I thought these were new tests. Yes, there are no new tests, just placed all the infinity checks inside their respective functions. Some of them had separate functions for the infinty tests, which weren't needed.

Also removed the TODO comments.

**Comment by srajangarg** [comment]
@leosartaj Anything else I should do?

**Comment by leosartaj** [comment]
You could change the PR title. Other than that +1.

**Comment by srajangarg** [comment]
@leosartaj Do you know why Travis isn't building? I need to get this merged in by 12 AM!

**Comment by leosartaj** [comment]
Don't be in a hurry. I will merge it when the tests pass. SymPy's test queue is known to be long. Expect it to be complete in next 24 hours.

**Pull request merged**